agenta-web/src/styles/tokens/antd-themeConfig.json

```diff
@@ -0,0 +1,2237 @@
+{
```

Comment:
This file is huge, for what it is.
Is it loaded on every request ?
Is there a minified/compressed version of this ?